ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -783,6 +804,7 @@ DFSS:
   - discharge_fan_lost_power_alarm
   - failed_discharge_fan_alarm
   - overload_discharge_fan_alarm
+  - local_override_alarm
```

Comment:
TBD on if this is specific enough (as there are other components that could be locally overridden) - chat w Trevor